wewatch-client/src/views/swiping/swiping-container.js

```diff
@@ -24,7 +24,9 @@ const SwipingContainer = (props) => {
   const [imageURL, setImageUrl] = useState(topTenMovies[index].image)
 
   const iterateMovie = (index) => {
-    setIndex(index + 1);
+    if(index !== 9) {
```

Comment:
instead of 9, should we just use topTenMovies.length?

Reply:
Good call. I'll fix it up. I think you mean `movies.length - 1` right?

Reply:
yes exactly!!